.bazelci/config.yaml

```diff
@@ -39,6 +39,15 @@ tasks:
       - "//cmake_defines:all"
     test_targets:
       - "//:tests_no_synthetic"
+  rbe_ubuntu1604_examples:
```

Comment:
🙌 Oh snap! 🙌

Reply:
I'm going to remove this from this PR and handle enabling this separately.